kernel/arch/m68k/GfPic.cpp

```diff
@@ -0,0 +1,74 @@
+#include <arch/m68k/GfPic.h>
+#include <debug/Log.h>
+#include <interrupts/Router.h>
+#include <memory/Vmm.h>
+
+namespace Npk
+{
+    constexpr size_t PicCount = 6;
+    constexpr size_t PicPins = 32;
+    constexpr uintptr_t PicRegsBase = 0xFF00'0000; //TODO: get from bootloader
```

Comment:
is the '  permitted here?

Reply:
it is, its ignored by the compiler. Its for making things for legible.